src/viam/robot/client.py

```diff
@@ -81,14 +81,17 @@ class RobotClient:
         from viam.robot.client import RobotClient
 
 
-        async def connect():
-            opts = RobotClient.Options.with_api_key(
-                # Replace "<API-KEY>" (including brackets) with your machine's API key
-                api_key='<API-KEY>',
-                # Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
-                api_key_id='<API-KEY-ID>'
-            )
-            return await RobotClient.at_address('<ADDRESS-FROM-THE-VIAM-APP>', opts)
+        async def connect() -> RobotClient:
+            opts = RobotClient.Options(
+                disable_sessions=True,
```

Comment:
Why are we suggesting disabling sessions?

Reply:
I don't know, I assume it was a copy paste error but just to double check @npentrel do you know if there was a reason for this? If not I will make a pr to edit on main docs page too.

Reply:
Where did you get this code from? Was that from [here](https://docs.viam.com/appendix/apis/robot/#establish-a-connection)?

Reply:
From the code sample you sent